mlink/mlink-functional-tests.groovy

```diff
@@ -0,0 +1,107 @@
+            steps {
+                script {
+                    currentBuild.displayName = "${params.MLINK_BRANCH}-${params.PSMDB}"
+                }
+            }
+        }
+        stage ('Run tests') {
+            matrix {
+                agent {
+                    label "${params.instance}"
+                }
+                axes {
+                    axis {
+                        name 'TEST'
+                        values 'replicaset', 'shard'
+                    }
+                }
+                stages {
+                    stage ('Run tests') {
+                        steps {
+                            withCredentials([string(credentialsId: 'olexandr_zephyr_token', variable: 'ZEPHYR_TOKEN')]) {
+                                sh """
+                                    docker kill \$(docker ps -a -q) || true
+                                    docker rm \$(docker ps -a -q) || true
+                                    docker rmi -f \$(docker images -q | uniq) || true
+                                    sudo rm -rf ./*
+                                    if [ ! -f "/usr/local/bin/docker-compose" ] ; then
+                                        if [ ${params.instance} = "docker-64gb-aarch64" ]; then
+                                            sudo curl -SL https://github.com/docker/compose/releases/download/v2.16.0/docker-compose-linux-aarch64 -o /usr/local/bin/docker-compose
+                                        else 
+                                            sudo curl -SL https://github.com/docker/compose/releases/download/v2.16.0/docker-compose-linux-x86_64 -o /usr/local/bin/docker-compose
+                                        fi
+                                        sudo chmod +x /usr/local/bin/docker-compose
+                                    fi
+                                """
+
+                                dir('psmdb-testing') {
+                                    git poll: false, branch: params.PSMDB_TESTING_BRANCH, url: 'https://github.com/Percona-QA/psmdb-testing.git'
+                                }
+
+                                withCredentials([string(credentialsId: 'GITHUB_API_TOKEN', variable: 'GIT_TOKEN')]) {
+                                    dir('percona-mongolink') {
+                                        git url: "https://x-access-token:${GIT_TOKEN}@github.com/Percona-Lab/percona-mongolink.git",
+                                                branch: params.MLINK_BRANCH,
+                                                poll: false
+                                    }
+                                }
+
+                                sh """
+                                    cd psmdb-testing/mlink
+                                    docker-compose build
+                                    docker-compose up -d
+                                    docker-compose run test pytest test_basic_sync_rs.py -v -s --junitxml=junit.xml -k ${TEST} || true
```

Comment:
Why do check test_basic_sync_rs.py only? When test are extended, there would be much more of those.
+
-k ${TEST} is unnecessary as mentioned above there is no need to divide on test type for now